CoreWidgetProvider/Helpers/GPUStats.cs

```diff
@@ -93,9 +101,18 @@ public void GetData()
 
             if (success)
             {
-                var sum = counters?.Sum(x => x.NextValue());
-                gpu.Usage = sum.GetValueOrDefault(0) / 100;
-                ChartHelper.AddNextChartValue(sum.GetValueOrDefault(0), gpu.GpuChartValues);
+                try
+                {
+                    var sum = counters?.Sum(x => x.NextValue());
```

Comment:
Nit: Add comment that `NextValue()` cant throw an exception (reason for the try catch) :)